acstools/polarization_tools.py

```diff
@@ -0,0 +1,414 @@
+            wfc:
+                filter: ['F475W', 'F606W']
+                pol0: [1.43, 1.33]
+                pol60: [1.47, 1.36]
+                pol120: [1.42, 1.30]
+            hrc:
+                filter: ['F330W']
+                pol0: [1.73]
+                pol60: [1.53]
+                pol120: [1.64]
+
+    Parameters
+    ----------
+    input_dict : dict
+
+    Examples
+    --------
+
+    To use the default values supplied in the acstools package (which come from the ACS
+    Data Handbook section 5.3):
+
+    >>> from acstools.polarization_tools import PolarizerTables
+    >>> tables = PolarizerTables.from_package_data()
+    >>> print(tables.wfc_efficiency)
+    filter  pol0  pol60  pol120
+    ------ ------ ------ ------
+     F475W 1.4303 1.4717 1.4269
+     F606W 1.3314 1.3607 1.3094
+     F775W 0.9965 1.0255 1.0071
+
+    To supply your own YAML file of the appropriate format:
+
+    >>> from acstools.polarization_tools import PolarizerTables
+    >>> tables = PolarizerTables.from_yaml('data/polarizer_tables.yaml')
+    >>> print(tables.wfc_transmission)
+    filter       t_para               t_perp             correction
+    ------ ------------------ ---------------------- ------------------
+    F475W 0.4239276798513098 0.00015240583841551956  1.000719276691027
+    F606W 0.5156734594049419 5.5908749369641956e-05  1.000216861312415
+    F775W 0.6040891283746557    0.07367364117759412 1.2777959654493372
+
+    >>> print(tables.wfc_transmission.meta['description'])
+    WFC filters use MJD corresponding to 2020-01-01. HRC filters use MJD corresponding to 2007-01-01.
+    """
+    def __init__(self, input_dict):
+
+        self.data = input_dict
+
+        self.wfc_transmission = Table(self.data['transmission']['wfc'],
+                                      names=('filter', 't_para', 't_perp', 'correction'),
+                                      meta=self.data['transmission']['meta'])
+
+        self.hrc_transmission = Table(self.data['transmission']['hrc'],
+                                      names=('filter', 't_para', 't_perp', 'correction'),
+                                      meta=self.data['transmission']['meta'])
+
+        self.wfc_efficiency = Table(self.data['efficiency']['wfc'], names=('filter', 'pol0', 'pol60', 'pol120'),
+                                    meta=self.data['efficiency']['meta'])
+
+        self.hrc_efficiency = Table(self.data['efficiency']['hrc'], names=('filter', 'pol0', 'pol60', 'pol120'),
+                                    meta=self.data['efficiency']['meta'])
+
+    @classmethod
+    def from_yaml(cls, yaml_file):
+        """
+        Read in a YAML file containing polarizer calibration data.
+
+        Parameters
+        ----------
+        yaml_file : str
+            Path to the YAML file containing the polarizer calibration information.
+
+        Returns
+        -------
+        `~acstools.polarizer_tools.PolarizerTables` object
```

Comment:
Need to reformat a bit for the API doc to show this part properly.
```suggestion
        pol_tables : `~acstools.polarizer_tools.PolarizerTables`
```